frontend/src/pages/_app.jsx

```diff
@@ -144,23 +120,13 @@ export default class extends App {
             }
           }
 
-          h2,
-          .modal-title {
+          h2 {
             font-size: 24px;
           }
 
-          .modal-body {
-            font-family: BentonSans;
-            font-weight: 300;
-          }
-
-          .page {
-            background: white;
-          }
-
           .app-wrapper {
             position: relative;
-            min-height: calc(100vh - 120px);
+            min-height: calc(100vh - 118px);
```

Comment:
118px is not enough to avoid vertical scroll on a page where there is not enough content, while 119px works fine